### PR TITLE
Fix PED bugs

### PR DESCRIPTION
**Pull request by jiayi1129**
- Add risk field checks for add and edit doctor commands
- Change `clear` command to `IWANTTOCLEAREVERYTHING`
- Remove proposed undo/redo implementation

### CODE REVIEW

**Review by ChuangZheQuan — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -176,14 +176,19 @@ Format: `toggle`
...
+Format: `IWANTTOCLEAREVERYTHING`
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+This command clears ALL patient, doctor and appointment records that you have on the application.
+Remember to do a backup in case you may need the data again. 
+</div>
 
 Examples:
-Typing `clear` in the removes all patient, doctor and appointment entries.
+Typing `IWANTTOCLEAREVERYTHING` in the removes all patient, doctor and appointment entries.
```
Comment (pualixue) [inline docs/UserGuide.md:191]: The "in the" sounds a bit weird, maybe can remove?

**Review by Haozeee — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (jiayi1129) [reply docs/UserGuide.md]: Good catch! Will remove 👍

**Pull request merged**